deepchecks/utils/features.py

```diff
@@ -150,15 +150,12 @@ def calculate_feature_importance(
 
     if force_permutation:
         if isinstance(dataset, pd.DataFrame):
-            permutation_failure = 'Cannot calculate permutation feature importance on a pandas Dataframe, using ' \
```

Comment:
why remove support for df?

Reply:
It's removing the behavior that if you mark "force_permutation" then it later looks for built-in feature importance. Instead, it fails.